superset-frontend/src/filters/components/GroupBy/types.ts

```diff
@@ -40,6 +40,8 @@ export type PluginFilterGroupByProps = PluginFilterStylesProps & {
   behaviors: Behavior[];
   data: DataRecord[];
   setDataMask: SetDataMaskHook;
+  setFocusedFilter: () => void;
+  unsetFocusedFilter: () => void;
```

Comment:
Simplification idea: we could move the `setDataMask`, `setFocusedFilter` and `unsetFocusedFilter` into a shared interface/type `PluginFilterHooks` in `filters/components/types.ts` and then just `&` them into `PluginFilterGroupByProps` and the others.

Reply:
Good point, done!